src/chem/monet_aeronet2ioda.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+#
+# (C) Copyright 2020 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+
+import sys
+import argparse
+import netCDF4 as nc
+import numpy as np
+from datetime import datetime, timedelta
+from monetio import aeronet
```

Comment:
I don't think we need to make a dependence on IODA-converters for MONET, but can we add some language somewhere, perhaps in the chem/README.md that says that this converter requires MONET to work? I suspect many of the users won't have it on their $PYTHONPATH

Reply:
Cory, I agree with your comment here. If such a lib is needed, it must indicate that this converter requires it in both script and README and make it convenient for the user.